src/main/java/xyz/nucleoid/extras/lobby/contributor/Contributor.java

```diff
@@ -0,0 +1,57 @@
+package xyz.nucleoid.extras.lobby.contributor;
+
+import java.util.UUID;
+
+import com.mojang.authlib.GameProfile;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.decoration.ArmorStandEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.nbt.NbtCompound;
+import net.minecraft.nbt.NbtHelper;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.text.Text;
+import xyz.nucleoid.extras.mixin.lobby.ArmorStandEntityAccessor;
+
+public record Contributor(String name, UUID minecraftUuid, NbtCompound statueNbt) {
+    public Text getName() {
+        return Text.literal(this.name);
+    }
+
+    public ItemStack createPlayerHead(MinecraftServer server) {
+        var playerHead = new ItemStack(Items.PLAYER_HEAD);
+        var nbt = playerHead.getOrCreateNbt();
+
+        var profile = new GameProfile(this.minecraftUuid, null);
+
+        if (profile.getId() != null && server != null) {
+            profile = server.getSessionService().fillProfileProperties(profile, false);
```

Comment:
Maybe off-thread? 😅 `SkullBlockEntity` has some (terrifying, but usable) code for this

Reply:
Fixed in 8114a2000a3b9c30b06ee2f7b81a21c5ce15578f